### ISSUE TITLE
Export timeshift records as files using FUSE

### ISSUE THREAD

**Issue by masnagam** [mirakc/mirakc#169]
Currently, there is no easy way to play timeshift records.
Implementing a frontend supporting the timeshift API is one of candidates, but it takes time.

As an alternative, we implement a feature to export timeshift records as files using [FUSE](https://en.wikipedia.org/wiki/Filesystem_in_Userspace).
We expect that this feature makes it easy for users to play records with media players like VLC and KODI which support MPEG2-TS.

The structure of the `timeshift` config will be changed like below:

```yaml
timeshift:
  mount: /path/to/mount-point
  recorders:
    nhk:
      ...
```

[zargony/fuse-rs](https://github.com/zargony/fuse-rs) is used for the filesystem implementation.

**Comment by masnagam** [comment]
Considering separation of the filesystem implementation from mirakc.

```
media players (running on a local machine)
  A
  |
  | export records as files
  |
mirakc-fs (running on a local machine)
  |
  | fetch data periodically via REST API
  V
mirakc (running on a remote server)
```

This is simpler than the original plan.
Separation of the function improves maintainability of both.

**Comment by masnagam** [comment]
Finally:

```
media players
  A
  |
samba/dlna (Docker container)
  | (scan media files)
  +==> files (accessible from the Docker host)
  | (expose using FUSE)
mirakc-timeshift-fs (Docker container)
  | (read)
  +==> timeshift records (ts and data files)
  | (write)
mirakc (Docker container)
```

**Comment by masnagam** [comment]
Implemented in 69f699c9901bf848e4d8bada6c77d3fcd7f9d938
